src/index-fn.ts

```diff
@@ -28,11 +32,13 @@ declare global {
   /* eslint-disable @typescript-eslint/no-empty-object-type */
   interface HTMLButtonElement extends PopoverToggleTargetElementInvoker {}
   interface HTMLInputElement extends PopoverToggleTargetElementInvoker {}
-  /* eslint-enable @typescript-eslint/no-empty-object-type */
+  /* eslint-enable @typescript-eslint/no-empty-interface */
 
   interface Window {
     ToggleEvent: ToggleEvent;
   }
 }
 
-export * from './popover.js';
+if (!isSupported()) {
+  apply();
+}
```

Comment:
I'm not sure I understand the changes in this file. The purpose of the `-fn` builds is for the polyfill _not_ to run automatically, but to allow users to run it conditionally on their own.

Reply:
Thanks for pointing this out! You're absolutely correct—the `-fn` build should not automatically apply the polyfill.